components/component-preview/src/component-preview/component-preview.tsx

```diff
@@ -21,20 +21,22 @@ import { syntaxHighlighterCustomStyles } from './syntax-highlighter.styles';
 
 export const ComponentPreview: FC<ComponentPreviewProps> = ({
   className,
-  component: Component,
   code,
-  onFullScreen,
-  themes,
   codeLanguage = 'tsx',
+  component: Component,
+  defaultThemeIndex,
+  onFullScreen,
   onLiveEdit,
   onThemeChange: onThemeChangeProp,
+  themes,
+  themeSwitcherLabel = 'Theme',
   wrapper: Wrapper = ({ children }) => <>{children}</>,
 }) => {
   // Styles
   const classes = useComponentPreviewStyles();
 
   // Theme switcher
-  const [themeKey, setThemeKey] = useState(themes?.[0].value);
+  const [themeKey, setThemeKey] = useState(themes?.[defaultThemeIndex ?? 0].value);
```

Comment:
Should we default defaultThemeIndex on init like the rest for consistency?

Reply:
👍